### PR TITLE
update roles doc

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: docs/roles.md [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
-**Some of the functions have changed after this document has written**
 
 The contracts in this repository define a number of different roles that have the power to change the way the contracts work. We list them here, with a short description of what an account with this role can do, and to which account this role will be assigned on deployment
 
-## `HatVaults.ProxyOwner`
+## `HatVaultsRegistry.owner`
 
-- set to `HATTimelockController`
-- can upgrade the proxy
-- can `transferOwnership` and `renounceOwnership`
```
Comment (ben-kaufman) [inline docs/roles.md:11]: Well that is possible no?

File: docs/roles.md [hunk 2]
```diff
@@ -77,9 +77,12 @@ The `owner` of hatvaults is HATs governance (currently a multisig, a DAO in the
 
 - set to governance multisig
 - can call `cancel` and cancel any pending operation
-- The following functions in HATVaults are **not** subject to a timelock:
-  - `addPool`
+
```
Comment (lirona) [inline docs/roles.md:80]:
```suggestion
- can call `setWithdrawalFee` on any vault - set the fee for withdrawals from the vault

```

File: docs/roles.md [hunk 3]
```diff
@@ -1,38 +1,32 @@
...
-- set to `HATTimelockController`
-- can upgrade the proxy
-- can `transferOwnership` and `renounceOwnership`
+- can call `setFeeSetter` set the feeSetter role
+- can call `setArbitrator` set the arbitrator role
+- can call `setWithdrawRequestParams` set time limits for withdrawal requests
+- can call `setClaimFee` set fee for submitting a vulnerability
+- can call `setChallengePeriod` 
+- can call `setChallengeTimeOutPeriod` 
+- can call `setWithdrawSafetyPeriod` set the amount of time during which claims can be submitted; during this time the vault users can not withdraw their funds. Must be less than 6 hours
```
Comment (lirona) [inline docs/roles.md:14]:
```suggestion
- can call `setWithdrawSafetyPeriod` to set the withdraw period and safety period, which are always interchanging. The safety period is time can submit claims for bounty payouts, and during which withdrawals are disabled and the bounty split cannot be changed.
```

File: docs/roles.md [hunk 4]
```diff
@@ -1,38 +1,32 @@
...
+## `HatVaultsRegistry.owner`
 
-- set to `HATTimelockController`
-- can upgrade the proxy
-- can `transferOwnership` and `renounceOwnership`
+- can call `setFeeSetter` set the feeSetter role
+- can call `setArbitrator` set the arbitrator role
+- can call `setWithdrawRequestParams` set time limits for withdrawal requests
+- can call `setClaimFee` set fee for submitting a vulnerability
+- can call `setChallengePeriod`
```
Comment (lirona) [inline docs/roles.md:12]:
```suggestion
- can call `setChallengePeriod` to set the time during which a claim can be challenged by the arbitrator
```

File: docs/roles.md [hunk 5]
```diff
@@ -77,9 +77,12 @@ The `owner` of hatvaults is HATs governance (currently a multisig, a DAO in the
...
-  - `addPool`
+
+
+## The following functions in HATVaults are **not** subject to a timelock:
   - `approveClaim`
+
   - `setPool`
-  - `setAllocPoints`
+  - `updateVaultInfo`
+  - `setAllocPoint`
```
Comment (lirona) [inline docs/roles.md:87]:
```suggestion
  - `setAllocPoint`
  - `setCommittee`
```

**Review by lirona — CHANGES_REQUESTED** [review]

File: docs/roles.md [hunk 6]
```diff
@@ -1,38 +1,32 @@
...
-## `HatVaults.ProxyOwner`
+## `HatVaultsRegistry.owner`
 
-- set to `HATTimelockController`
-- can upgrade the proxy
-- can `transferOwnership` and `renounceOwnership`
+- can call `setFeeSetter` set the feeSetter role
+- can call `setArbitrator` set the arbitrator role
+- can call `setWithdrawRequestParams` set time limits for withdrawal requests
+- can call `setClaimFee` set fee for submitting a vulnerability
```
Comment (lirona) [inline docs/roles.md:11]:
```suggestion
- can call `setClaimFee` to set the fee for logging a claim for a bounty in any vault.
```

File: docs/roles.md [hunk 7]
```diff
@@ -1,38 +1,32 @@
...
+- can call `setChallengePeriod` 
+- can call `setChallengeTimeOutPeriod` 
+- can call `setWithdrawSafetyPeriod` set the amount of time during which claims can be submitted; during this time the vault users can not withdraw their funds. Must be less than 6 hours
+- can call `setHatVestingParams` set vesting paramaters for bounty paid in hats
+- `setMaxBountyDelay` - Set the timelock delay for setting the max bounty
+- can call `setRouterWhitelistStatus` add or remove address from the whitelist of routers that can be used for token swapping
+- can call `swapBurnSend` and swap, burn, and send the vauit's token that are earmarked for payout (after approveClaim)
```
[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply docs/roles.md:18]: why?

[the remark this answers is not in the document]
Comment (lirona) [reply docs/roles.md:18]: because swapBurnSend is run by the owner of the vault and not the registry

File: docs/roles.md (hunk 3 above)
Comment (lirona) [reply docs/roles.md:14]: more accurate because it sets both

**Pull request merged**